std_msgs/Header header
# desired position
float64 x
float64 y
float64 z
float64 p2p_traj_duration
# desired ZYZ
float64 alpha
float64 beta
float64 gamma
# info
bool accepted
float64 elapsed_time

